tests/_files/test_audio.py

```diff
@@ -71,7 +72,7 @@ def test_creation(self, audio):
         assert audio.file_unique_id
 
     def test_expected_values(self, audio):
-        assert audio.duration == self.duration
+        assert audio._duration == self.duration
```

Comment:
I would prefer it if we don't rely on protected members in the test suite. Does `audio.duration == self.duration` not suffice? Or does the `PTB_TIMEDELTA` fixtire inferfere in concurrent testing?